inc/json-ld-namespace.php

```diff
@@ -92,9 +92,14 @@ function singular( array $meta, array $context ) : array {
 	$meta['datePublished'] = get_the_date( 'c', $context['object'] );
 	$meta['dateModified'] = get_the_modified_date( 'c', $context['object'] );
 	$meta['mainEntityOfPage'] = get_the_permalink( $context['object_id'] );
-	$meta['author'] = [
-		get_person( get_user_by( 'id', $context['object']->post_author ) )
-	];
+	
+	// Post author is only set for post types that support it.
+	if ( post_type_supports( $context['object']->post_type, 'author' ) && isset( $context['object']->post_author ) ) {
```

Comment:
Not a blocker just a question: Is it better to use `! empty` here instead of `isset`, just in case we have a bunch of posts that have `post_author` `0` which is what currently happens. If it's fine to use `isset` here then I'll approve this PR.

Reply:
That's a good shout, thanks @stuartshields